test/utils/wasm_engine.hpp

```diff
@@ -1,6 +1,5 @@
 #pragma once
 
-#include "types.hpp"
```

Comment:
Oh this is actually needed for `fizzy_bytes` too, but is only visible if trying to compile wabt/wasm3.